app/views/bento/index.json.jbuilder

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
```

Comment:
I started with [the json builder in blacklight](https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/index.json.jbuilder) and trimmed what we don't need.